services/catalog/src/simcore_service_catalog/core/settings.py

```diff
@@ -102,7 +102,7 @@ def loglevel(self) -> int:
     debug: bool = False  # If True, debug tracebacks should be returned on errors.
 
     # background task
-    background_task_rest_time: PositiveInt = 5
+    background_task_rest_time: PositiveInt = 20
```

Comment:
I could not figure out why this interval is so "low". Why is it important to update so aggressively?

Reply:
it is not. I will change it that until it runs one it is low, and then increase it to much more.

Reply:
on 60 now